### PR TITLE
Add popup colors

### PR DESCRIPTION
**Pull request by johngruen**
This is done over-top of my image cleanup PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "firefoxcolor",
   "description": "Theming experiment for Firefox Quantum and beyond.",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (johngruen) [inline package.json]: bumped version to pick up add-on changes

File: src/extension/background.js [hunk 2]
```diff
@@ -230,6 +230,14 @@ const applyTheme = ({ theme }) => {
     newTheme.colors.tab_loading = colorToCSS(theme.colors.tab_line);
   }
 
+  if (!theme.colors.hasOwnProperty("popup")) {
```
Comment (johngruen) [inline src/extension/background.js]: If users have previously built themes, we should take a guess at the right colors for popups and popup text

File: src/lib/themes.js [hunk 3]
```diff
@@ -116,17 +120,20 @@ export const normalizeThemeBackground = background =>
   bgImages.keys().includes(background) ? background : null;
 
 // Utility to ensure normal & consistent colors
-export const normalizeThemeColor = (data, defaultColor) =>
-  makeTinycolor(data || defaultColor).toRgb();
+export const normalizeThemeColor = (name, data, defaultColor) => {
```
Comment (johngruen) [inline src/lib/themes.js]: This is getting called independently of normalizeThemeColors in the redux store causing alpha values to leak. I moved the alpha check into this function to resolve

File: src/lib/themes.js [hunk 4]
```diff
@@ -116,17 +120,20 @@ export const normalizeThemeBackground = background =>
...
 
 export const normalizeThemeColors = (colors = {}) => {
   const out = {};
   const { colors: defaultColors } = defaultTheme;
   Object.keys(defaultColors).forEach(name => {
-    const color = normalizeThemeColor(colors[name], defaultColors[name]);
-    if (!colorsWithAlpha.includes(name)) {
-      delete color.a;
-    }
+    const matchedColor = colors[name] || colors[fallbackColors[name]];
```
Comment (johngruen) [inline src/lib/themes.js]: also, fallback if no color is available

**Comment by meandavejustice** [comment]
needs rebase @johngruen

**Comment by lmorchard** [comment]
LGTM - theme schema validation, tests, works well, looks good :+1:

**Pull request merged**